ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/run-details-bar.tsx

```diff
@@ -130,6 +135,23 @@ export const RunDetailsBar = ({ className }: RunDetailsBarProps) => {
           <Label className='font-semibold'>Created at:</Label>
           <TimestampWithUTC timestamp={ortRun.createdAt} />
         </div>
+        {ortRun.userDisplayName && (
```

Comment:
As mentioned in the daily, I'd have a slight preference for moving this above "Created at" to keep the two time-related properties "Created at" and "Duration" next to each other.